website/addons/owncloud/static/owncloudUserConfig.js

```diff
@@ -0,0 +1,226 @@
+
+
+var ko = require('knockout');
+var $ = require('jquery');
+var Raven = require('raven-js');
+var bootbox = require('bootbox');
+require('js/osfToggleHeight');
+
+var language = require('js/osfLanguage').Addons.owncloud;
+var osfHelpers = require('js/osfHelpers');
+var addonSettings = require('js/addonSettings');
```

Comment:
The only thing this is used for is declaring `ExternalAccount` --  why not just 
```javascript
var ExternalAccount = require('js/addonSettings').ExternalAcccount
```

(disregard this if you implement the refactor mentioned below)